x-pack/plugins/cloud_security_posture/server/saved_objects/migrations/csp_rule_template.ts

```diff
@@ -28,7 +28,7 @@ function migrateCspRuleTemplatesToV840(
       muted,
       metadata: {
         ...metadata,
-        benchmark: { ...benchmark, id: 'cis_k8s', rule_number: '' },
+        benchmark: { ...benchmark, id: 'cis_k8s' },
         impact: metadata.impact || undefined,
         default_value: metadata.default_value || undefined,
         references: metadata.references || undefined,
```

Comment:
👍 

We need to check that indeed nothing needs to be added to 8.7 migration

Reply:
since we can't use a default value here that is suitable to all older findings, the only value that can be passed here is `undefined`. the previous of using an empty string is ok(ish) but undefined is more accurate to the state. it means that the table will show an empty cell for older findings in this specific `rule number` table column